_includes/_guides/_pool_crunching.htm

```diff
@@ -0,0 +1,135 @@
+{% include _guides/_boinc_introduction.htm %}
+
+<section id="section_solo_steps" class="container">
+  <h2 class="echo-header">
+      <span class="echo" aria-hidden="true">Step-by-Step Guide</span>
+      Step-by-Step Guide
+  </h2>
+
+  <div class="steps">
+      <section class="step">
+          <header>
+              <img src="/assets/img/ic-guide-install-boinc.svg" alt="">
+              <h3>
+                  <span class="echo" aria-hidden="true">01</span>
+                  <strong>Step 1.</strong> Install BOINC
+              </h3>
+          </header>
+          <div class="step-body">
+              <p>
+                  You must install the BOINC client and select one or more BOINC
+                  projects. Go to the <a href="https://boinc.berkeley.edu/">BOINC
+                  website</a>, download and install BOINC for any computer that
+                  you want to run BOINC on. If you don't choose projects that use
+                  VirtualBox, the BOINC+VirtualBox package is not needed.
```

Comment:
The double negative here is a bit confusing. Perhaps something like this would be better 
```suggestion
                  you want to run BOINC on. The BOINC+VirtualBox package is 
                  only needed if you choose projects that use VirtualBox.
```